samples/scripts/setup.ps1

```diff
@@ -3,8 +3,8 @@ Write-Output "Sign in to Microsoft 365..."
 npx -p @pnp/cli-microsoft365 -- m365 login --authType browser
 
 # create AAD app
-Write-Output "Creating AAD app..."
-$appId = npx -p @pnp/cli-microsoft365 -- "m365 aad app add --name graph-developer-proxy-samples --multitenant --redirectUris http://localhost:3000/withsdk.html,http://localhost:3000/nosdk.html --apisDelegated https://graph.microsoft.com/User.Read.All,https://graph.microsoft.com/Presence.Read.All --grantAdminConsent --platform spa --query appId -o text"
+Write-Output "Creating Entra ID app..."
```

Comment:
It's actually `Entra app`. You only use `Entra ID` when referring to the product itself, but not its features

Reply:
Good catch. Naming is hard 😅